programs/mango-v4/src/state/oracle.rs

```diff
@@ -425,9 +425,11 @@ fn oracle_state_unchecked_inner<T: KeyedAccountReader>(
             };
 
             let clmm_price = if inverted {
-                let inverted_price = I80F48::from_num(sqrt_price * sqrt_price);
-                I80F48::ONE.checked_div(inverted_price).unwrap()
+                let sqrt_price = f64::from_fixed(U64F64::from_bits(whirlpool.sqrt_price));
```

Comment:
minor, for consistency: we've typically used `.to_num::<f64>()` for this conversion - prefer to use that here if that's allowed